docs/data-sources/cdn_domain_certificates.md

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: Content Delivery Network (CDN)
+---
+
+# huaweicloud_cdn_domain_certificates
+
+Use this data source to get the domains bound to HTTPS certificate of CDN.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_cdn_domain_certificates" "test" {
+  name = "abcdetest.huaweicloud.com"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `domain_name` - (Optional, String) Specifies the name of the acceleration domain.
```

Comment:
Please confirm whether the field name is `name` or `domain_name`.
Check carefully whether the naming of other fields is compliant.